core/src/test/scala/com/nvidia/spark/rapids/BaseTestSuite.scala

```diff
@@ -68,6 +68,11 @@ class BaseTestSuite extends FunSuite with BeforeAndAfterEach with Logging {
       "Spark340 does not parse the eventlog correctly")
   }
 
+  protected def shouldSkipUnsupportedExprForSparkLT340(): (Boolean, String) = {
```

Comment:
Does `SparkLT340` mean `greater than Spark340` or `less than Spark340`? If it refers to `less than Spark340`, then does the function mean `shouldSkipUnsupportedExprForLessThanSpark340`, but we actually want to skip >= Spark340.

Reply:
renamed the method